_data/banking.yml

```diff
@@ -972,6 +972,14 @@ websites:
       facebook: SimpliiFinancial
       twitter: SimpliiFin
 
+    - name: SoFi
+      url: https://www.sofi.com/
+      img: sofi.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
```

Comment:
I don't see anything about software 2FA on the doc page you provided.

Reply:
@Carlgo11 - that blog post is the closest thing they have to any documentation on this, and I'm guessing they've added software since then since I see it in my account.

Would it be better to have no documentation link than an incomplete one?

![Screenshot 2019-07-31 07 36 42](https://user-images.githubusercontent.com/96935/62212422-52480b00-b366-11e9-8737-092de8d33efd.png)